### DISCUSSION TITLE
Metadata dataset: Resource Locator / Check that at least one  resource locator is defined.

### DISCUSSION THREAD

**Discussion by fxprunayre** [INSPIRE-MIF/helpdesk-validator#1251] [Ideas]
"A Resource locator linking to the service(s) providing online access to the described data set or data set series shall be given, if such online access is available." does not mean that **all** `gmd:onLine/gmd:URL` have to be "valid" URLs.

Report about the expression used to check the URL as it is a bit
restricted vision of what an URL is eg. file://, jdbc://, PRL starting
with // are not considered valid.

Proposal:
* Check that at least one URL is following the expected pattern 
* If none found, report error with the pattern used

Changes available in https://github.com/inspire-eu-validation/ets-repository/pull/315/commits/144aaad043435388c0e5f172d7bc8eea06622dfe

When no valid URL provided, then message is now:
```
XML document 'file.xml', record 'c651bc41-d5da-49de-ba47-c51eabbb2007': No online source
 element found in the record following the rule 
'(http://|https://|ftp://)([a-z0-9]{1})((\.[a-z0-9-])|([a-z0-9-]))*\.([a-z0-9-]{1,4})(/?)'.
```

Relates to https://github.com/inspire-eu-validation/ets-repository/pull/315

**Comment by michellutz** [comment]
I am not sure about this change proposal. Given that there are on-going discussions in the MIG about using the resource locator element more extensively, I think this should be discussed with the 2017.4 sub-group.

**Comment by jmrubio82** [comment]
Hi,
I understand this is still not yet solved or fully addressed, considering our current discussions about the use of resource locator and the alternative linking approach, but has there been any advance on this particular issue?

**Comment by fabiovinci** [comment]
Dear all,
since this issue requires the development of new features that are currently unplanned, it will be moved to the related discussion section.
